.github/workflows/release.yaml

```
@@ -47,56 +47,110 @@ jobs:
 
       - name: Download sagemaker artifacts by commit ID
         run: |
-          # Find artifact by name using curl
-          ARTIFACT_ID=$(curl -L \
+          # Download build artifact
+          BUILD_ARTIFACT_ID=$(curl -L \
             -H "Accept: application/vnd.github+json" \
             -H "Authorization: Bearer $GH_TOKEN" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
             "https://api.github.com/repos/$REPOSITORY/actions/artifacts?name=$COMMIT_SHA-code-editor-sagemaker-server-build" \
             | jq -r ".artifacts[0].id // empty")
           
-          if [ -n "$ARTIFACT_ID" ]; then
+          # Download src artifact
+          SRC_ARTIFACT_ID=$(curl -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer $GH_TOKEN" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            "https://api.github.com/repos/$REPOSITORY/actions/artifacts?name=$COMMIT_SHA-code-editor-sagemaker-server-src" \
+            | jq -r ".artifacts[0].id // empty")
+          
+          if [ -n "$BUILD_ARTIFACT_ID" ]; then
+            curl -L -H "Authorization: Bearer $GH_TOKEN" \
+              "https://api.github.com/repos/$REPOSITORY/actions/artifacts/$BUILD_ARTIFACT_ID/zip" \
+              -o build-artifact.zip
+            unzip build-artifact.zip
+          else
+            echo "No build artifact found matching pattern: $COMMIT_SHA-code-editor-sagemaker-server-build"
+            exit 1
+          fi
+          
+          if [ -n "$SRC_ARTIFACT_ID" ]; then
             curl -L -H "Authorization: Bearer $GH_TOKEN" \
-              "https://api.github.com/repos/$REPOSITORY/actions/artifacts/$ARTIFACT_ID/zip" \
-              -o artifact.zip
-            unzip artifact.zip
+              "https://api.github.com/repos/$REPOSITORY/actions/artifacts/$SRC_ARTIFACT_ID/zip" \
+              -o src-artifact.zip
+            unzip src-artifact.zip
           else
-            echo "No artifact found matching pattern: $COMMIT_SHA-code-editor-sagemaker-server-build"
+            echo "No src artifact found matching pattern: $COMMIT_SHA-code-editor-sagemaker-server-src"
             exit 1
           fi
```

Nit: GitHub CLI is available, it's more concise to write
```suggestion
          gh run download --name "$COMMIT_SHA-code-editor-sagemaker-server-build.zip" --name "$COMMIT_SHA-code-editor-sagemaker-server-src.zip"
          unzip "$COMMIT_SHA-code-editor-sagemaker-server-build.zip"
          unzip "$COMMIT_SHA-code-editor-sagemaker-server-src.zip"
```